scripts/gzip-all.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+const gzipAll = require('gzip-all');
+
+var walkSync = function(dir, filelist) {
+    if (dir[dir.length - 1] != path.sep) dir = dir.concat(path.sep);
+
+    var fs = require("fs"),
+        files = fs.readdirSync(dir);
+    filelist = filelist || [];
```

Comment:
Could be moved to function argument `function(dir, filelist = [])`

Reply:
Done.